Data-Transfer-ReactJS-Dashboard/src/pages/TransferQueue.js

```diff
@@ -0,0 +1,67 @@
+import './TransferQueue.css'
+const TransferQueue = () => {
+    return (
+      <div className = "TransferQueue">
+        <p className="TransferQueueHeading">Tranfer Queue</p>
+        <div className='Transfer1'>
+          <p className='Transfer1Value'>Transfer 1</p>
+          <div className='eclipse1'>
+            <p className='value1'>1</p>
+          </div>
+          <p className='Transfer1Status'>Ongoing</p>
+          <div className='Transfer1Fields'>
+            <p className='TotalSize'>Total Size</p>
+            <p className='TotalSizeValue'>5 PB</p>
+            <p className='DateStarted'>Date Started</p>
+            <p className='DateStartedValue'>2 June 2022</p>
+            <p className='EstimatedCompletion'>Estimated Completion</p>
+            <p className='EstimatedCompletionValue'>2 August 2022</p>
+            <div className='viewDataCatalogueButton'>  
+              <p className='viewDataCatalogue'>View Data Catalogue</p>
+            </div>
+          </div>
+        </div>
+        <div className='Transfer1Line'></div>
+        <div className='Transfer2'>
+          <p className='Transfer2Value'>Transfer 2</p>
+          <div className='eclipse2'>
+            <p className='value'>2</p>
+          </div>
+          <p className='Transfer2Status'>Not Started</p>
+          <div className='Transfer2Fields'>
+            <p className='TotalSize'>Total Size</p>
+            <p className='TotalSizeValue'>5 PB</p>
+            <p className='DateStarted'>Date Started</p>
+            <p className='DateStartedValue'>-</p>
+            <p className='EstimatedCompletion'>Estimated Completion</p>
+            <p className='EstimatedCompletionValue'>-</p>
+            <div className='viewDataCatalogueButton'>  
+              <p className='viewDataCatalogue'>View Data Catalogue</p>
+            </div>
+          </div>
+        </div>
+        <div className='Transfer2Line'></div>
+        <div className='Transfer3'>
```

Comment:
Why hard coding?

Reply:
This Transfer Queue page as of now is not yet implemented in the back end. So Hardcoded the values to make this page look good. In the future, once the backend implements this, we will make this page dynamic.

Reply:
You can still utilize a json file or some other mechanism to have this setup. Just a thought.